src/main/java/reposense/commits/CommitResultAggregator.java

```diff
@@ -39,60 +44,74 @@ public static CommitContributionSummary aggregateCommitResults(
     }
 
     private static Map<Author, Float> calcAuthorContributionVariance(
-            Map<Author, List<AuthorDailyContribution>> intervalContributionMaps) {
+            Map<Author, List<AuthorDailyContribution>> intervalContributionMaps, Date startDate, Date lastDate) {
         Map<Author, Float> result = new HashMap<>();
         for (Author author : intervalContributionMaps.keySet()) {
             List<AuthorDailyContribution> contributions = intervalContributionMaps.get(author);
-            result.put(author, getContributionVariance(contributions));
+            result.put(author, getContributionVariance(contributions, startDate, lastDate));
         }
         return result;
     }
 
-    private static float getContributionVariance(List<AuthorDailyContribution> contributions) {
+    private static float getContributionVariance(List<AuthorDailyContribution> contributions,
+            Date startDate, Date lastDate) {
         if (contributions.size() == 0) {
             return 0;
         }
         //get mean
         float total = 0;
+        long totalDays = (lastDate.getTime()
+                - getStartOfDate(startDate).getTime()) / DAYS_IN_MS + 1;
```

Comment:
Make it a higher level break. Breaking in the middle of `(lastDate.getTime() - getStartOfDate(startDate).getTime())` is considered a lower level break because they're treated as a single element in the calculation against the `/ DAYS_IN_MS`.

Do follow the example i gave earlier.